src/custodian/src/custodian_monitor.erl

```diff
@@ -10,20 +10,16 @@
 % License for the specific language governing permissions and limitations under
 % the License.
 
-{application, custodian,
- [
-  {description, "in your cluster, looking after your stuff"},
-  {vsn, git},
-  {registered, []},
-  {applications, [
-                  kernel,
-                  stdlib,
-                  couch_log,
-                  config,
-                  couch_event,
-                  couch,
-                  mem3
-                 ]},
-  {mod, { custodian_app, []}},
-  {env, []}
- ]}.
+-module(custodian_monitor).
+
+
+-callback send_missing_db_alert(DbName :: binary()) ->
+    Output :: string().
```

Comment:
I'd encode the fact that we don't use the result of these functions, we call them only for their implementation. so perhaps insist on `false` atom in all cases or say `any()` and document that the result of the function is ignored.

Reply:
Done in fixup